### PR TITLE
profiles/base/use: add `minimal` useflag to `dev-lang/perl`

### PR DESCRIPTION
**Pull request by tormath1**
Try to use `minimal` version of `Perl`.

## Testing done

jenkins (:large_blue_circle:): http://jenkins.infra.kinvolk.io:8080/job/os/job/manifest/3603/cldsv/

To be merged with: https://github.com/flatcar-linux/portage-stable/pull/215

### CODE REVIEW

**Review by krnowak — APPROVED** [review]

**Comment by tormath1** [comment]
@krnowak I missed the useflag for the SDK so it broke - I restarted a build !

**Comment by jepio** [comment]
It would be good to test if minimal is still enough to build torcx-zfs (https://github.com/lorenz/torcx-zfs) as the zfs build system uses a lot of perl to test the kernel configuration.

**Comment by tormath1** [comment]
@krnowak @jepio one difference since the latest approval; `minimal` has been added to `BOOTSTRAP_USE`, CI is OK and `zfs` builds correctly.

File: profiles/coreos/targets/sdk/package.use [hunk 1]
```diff
@@ -33,3 +33,6 @@ cross-aarch64-cros-linux-gnu/gdb python_single_target_python3_6 -python_single_t
 
 # Enable gssapi for SDK
 net-dns/bind-tools           gssapi
+
+# Install `perl` with a minimal set of dependencies
+dev-lang/perl minimal
```
Comment (krnowak) [inline profiles/coreos/targets/sdk/package.use:38]: Is this needed if you already added it to coreos/base profile? Both arm64 and amd64 sdk profiles already inherit from coreos/base profile.

**Review by krnowak — COMMENTED** [review]
One question, otherwise looks alright.

File: profiles/coreos/targets/sdk/package.use (hunk 1 above)
Comment (tormath1) [reply profiles/coreos/targets/sdk/package.use:38]: ok good to know - so `profiles/sdk` is only to override things from `profiles/base`.

**Comment by krnowak** [comment]
There's an all-full build for this PR and for the texinfo update PR: http://localhost:9091/job/os/job/manifest/3654

**Review by krnowak — APPROVED** [review]
CI passed, let's get this in.

**Review by sayanchowdhury — APPROVED** [review]
:shipit:

**Pull request merged**